app/controllers/items_controller.rb

```diff
@@ -17,13 +18,18 @@ def show
   end
 
   def new
-    @item = Item.new(status: 'Available')
+    @item_type = ItemType.find ( params[:item_type_id])
+    @item = Item.new(
+            status: 'Available',
+            item_type_id: params[:item_type_id])
+
     @page_title = "New Item"
   end
 
   def edit
     @item = Item.find(params[:id])
-    @page_title = "Editting Item #{@item.name}"
+    @item_type = @item.item_type
```

Comment:
again, This feels wrong to me. This page is about items and the item has the ItemType in it so, i'd just ask the item for its item type in the view when needed instead of making a second var for it. 

Also, removing these unnecessary 2nd vars means you have less things to test the presence of in a controller  / integration test.